src/main/java/umc/spring/web/controller/MissionController.java

```diff
@@ -0,0 +1,57 @@
+package umc.spring.web.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.Parameters;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import umc.spring.Vaildator.annotation.CheckPage;
+import umc.spring.apiPayload.ApiResponse;
+import umc.spring.service.MissionService.MissionQueryService;
+import umc.spring.web.controller.dto.MissionDTO.MissionRequestDTO;
+import umc.spring.web.controller.dto.MissionDTO.MissionResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping("/mission")
+public class MissionController {
+
+    private final MissionQueryService missionService;
+
+    @PostMapping("/add")
+    public ApiResponse<MissionResponseDTO.JoinResultDTO> addMission(@RequestBody @Valid MissionRequestDTO.JoinAddMissionDTO request){
```

Comment:
한 줄의 길이가 가능한 줄여서, 코드 가독성을 높여주는 것도 중요한 거 같아요!!! 나중에 프로젝트하실 때  팀 컨벤션에 맞춰서 코드를 작성하면 좋을 거 같네요!!
```suggestion
    public ApiResponse<MissionResponseDTO.JoinResultDTO> addMission(
        @RequestBody @Valid MissionRequestDTO.JoinAddMissionDTO request
    ){
```